mojave/items/misc/fluff.dm

```diff
@@ -90,10 +90,11 @@
 /obj/item/ms13/fluff/microscope/screwdriver_act_secondary(mob/living/user, obj/item/weapon)
 	user.show_message(span_notice("You begin disassembling \the [src] into scrap."), MSG_VISUAL)
 	if(do_after(user, 8 SECONDS, target = src, interaction_key = DOAFTER_SOURCE_DECON))
+		var/drop_location = user.drop_location()
```

Comment:
Why not define this on the fluff base type?

Reply:
good question

Reply:
not my problem tho imma be real